paket.dependencies

```diff
@@ -1,7 +1,10 @@
-storage:none
-source https://www.nuget.org/api/v2
-
-nuget FSharp.Core >= 4 redirects: force
-nuget protobuf-net
-nuget Expecto
-nuget FsCheck
\ No newline at end of file
+storage:none
+source https://www.nuget.org/api/v2
+
+nuget FSharp.Core >= 4 redirects: force
+nuget protobuf-net >= 3.0
+nuget FsCheck >= 2.0 < 3.0
+nuget Expecto >= 9.0 < 10.0
+nuget Expecto.FsCheck >= 9.0 < 10.0
+nuget YoloDev.Expecto.TestSdk >= 0.13 < 0.14
+nuget Microsoft.NET.Test.Sdk
```

Comment:
You shouldn't need to bring this in via Paket - it usually gets imported via the ProjectType in the *.fsproj file I thought.

Reply:
If you mean `Microsoft.NET.Test.Sdk` part to make `dotnet test` work specifically, it was different a few years ago (if I remember correctly), but seems to have stabilized at importing `Microsoft.NET.Test.Sdk` as any other nuget package now --- at least it works from netcore3.1 to net7.0